### ISSUE TITLE
The Dependency System.XML could not be resolved - ASP.NET Core 1 - RC2

### ISSUE THREAD

**Issue by YannisMarios** [IdentityServer/IdentityServer4#99]
Hi,

I get this error on ASP.NET Core 1 RC2:

The Dependency System.XML could not be resolved	

Please see images attached

Thank you,

![screenshot_1](https://cloud.githubusercontent.com/assets/11289298/15465606/0f7671f4-20de-11e6-9d26-76bfce3e0a55.png)
![screenshot_2](https://cloud.githubusercontent.com/assets/11289298/15465605/0f754932-20de-11e6-9267-e05c2a3f0a45.png)
![screenshot_3](https://cloud.githubusercontent.com/assets/11289298/15465604/0f6ec4a4-20de-11e6-8342-edc7f7149c4a.png)

**Comment by bburman** [comment]
I had the same problem. It looks like you are targeting dnx451 in your frameworks section. Apparently the newest version of IdentityServer4 is currently only supporting netstandard1.5, which would be .NET 4.6.2 and above.

**Comment by YannisMarios** [comment]
Thank you!

Your tip helped me!

**Comment by bburman** [comment]
@YannisMarios You are welcome! I'm glad it helped!

**Comment by YannisMarios** [comment]
I am now getting:

System.IO.FileNotFoundException
Could not load file or assembly 'System.Runtime, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.

Any ideas?

Thank you.


![image](https://cloud.githubusercontent.com/assets/11289298/15584058/78a28cc6-2381-11e6-8300-12418d2c88f8.png)

**Comment by leastprivilege** [comment]
i updated all the packages - give it a go and see if that works better for you (and open a new issue if you encounter problems)

**Comment by torrequinn** [comment]
I know this is older and you seem to have resolved your issue, but...

I ran into the same error message. What I found is that if you add the reference to System.Xml by right clicking on references and seeking out the assemply and adding it; The system adds an entry to your project.json for "System.XML" (mucth like you referenced above).

The reference did not resolve, so I changed the case in the json to "System.Xml" (note case difference), did a rebuild and it worked fine.
